packages/react-resizable-panels/src/Panel.test.tsx

```diff
@@ -377,7 +377,7 @@ describe("PanelGroup", () => {
       });
     });
 
-    test("should throw if default size is less than 0 or greater than 100", () => {
+    test.skip("should throw if default size is less than 0 or greater than 100", () => {
```

Comment:
this runs into some infinite loop or smth and makes the test hang - I'm still investigating what's up with this

Reply:
Ok, so this version pass:
```ts
    test("should throw if default size is less than 0 or greater than 100", () => {
      expectWarning(
        "Invalid layout total size: -1%. Layout normalization will be applied."
      );
      expectWarning("default size should not be less than 0");

      act(() => {
        root.render(
          <PanelGroup direction="horizontal">
            <Panel defaultSize={-1} />
          </PanelGroup>
        );
      });

      act(() => {
        root.render(
          <PanelGroup direction="horizontal">
            <Panel defaultSize={101} />
          </PanelGroup>
        );
      });
    });
```

but clearly it doesn't do what the test title says. So something is off here and has to be fixed - I'll leave that to you ;p

Reply:
Works for me when I uncommented the `skip` 🤷🏼